### PR TITLE
Add council page

### PR DESCRIPTION
**Pull request by sdruskat**
This PR does the following things:

- Hide the 2020 workshop link from the main navigation menu
- Add a page for the Intl RSE Council and add a menu item to the main navigation
- Provide a custom key `hidetitle` for use in YAML page metadata to hide the title on pages
- Provide a custom YAML page metadata key `nav` to define a heading to be used in the page instead of the default `page.title`
- Add a logo for the Council

### CODE REVIEW

**Comment by bast** [comment]
Very minor suggestion: to (auto)wrap long text lines to make review of future changes easier with smaller future diffs.

**Comment by sdruskat** [comment]
Thanks @bast, done in https://github.com/RSE-leaders/researchsoftware.org/pull/24/commits/e28247e758f6b08e3ccd584a6ffd70a7f96e9130 :).

**Comment by sdruskat** [comment]
@all-contributors please add @bast for review.

File: council.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+charter (currently being drafted). Further national or multinational RSE 
+associations can join the Council once they pass a threshold for establishment 
+that is defined in the charter.
+
+To maintain openness towards the community and facilitate further knowledge
+exchange and support, representatives of nascent communities can join Council 
+meetings as observing participants, ask questions and discuss issues at the 
+end of meetings.
+
+To get in touch with the Council, write an email to the public mailing list at
```
Comment (sdruskat) [inline council.md:28]:
```suggestion
To get in touch with the Council, write an email to
```

Remove mailing list mention.

File: council.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+| de-RSE  | Frank Löffler |
+| BE-RSE  | Alexander Botzki |
+| BE-RSE  | Johan Philips |
+| NL-RSE  | Anna-lena Lamprecht |
+| NL-RSE  | Ben van Werkhoven |
+| RSE-AUNZ    | Justin Baker |
+| RSE-AUNZ    | Nicholas May |
+| Nordic-RSE  | Anne Claire Fouilloux |
+| Nordic-RSE  | Radovan Bast |
+| US-RSE  | Dan Katz |
```
Comment (danielskatz) [inline council.md:50]:
```suggestion
| US-RSE  | Daniel S. Katz |
```

**Pull request merged**